Makefile

```diff
@@ -21,12 +27,20 @@ docker-shell:
 	$(run_docker) /bin/bash
 
 .PHONY: test-sqlalchemy
-test-sqlalchemy:
+test-sqlalchemy: run-joplin-sqlalchemy
 	$(run_docker) /bin/bash -c 'export && cd /app/stac_fastapi/sqlalchemy/tests/ && pytest'
 
 .PHONY: test-pgstac
-test-pgstac: pgstac-install
-	$(run_docker) /bin/bash -c 'export && cd /app/stac_fastapi/pgstac/tests/ && pytest'
+test-pgstac:
```

Comment:
Should this also have a dependency on a `run-joplin-pgstac` target?

Reply:
I cleared my Docker caches using:

```
docker system prune
docker volume prune
```
and then ran the tests using:
```
make run-database
make test-pgstac
```
and all of the tests succeeded, which made me think that it wasn't relying on that data. We could add that as a dependency just in case we write tests later that need it.

Reply:
Interesting - seems like the pgstac test were written to not rely on the dev datasets, which I think is a better approach. Thanks for clearing that up.